openml/flows/sklearn_converter.py

```diff
@@ -0,0 +1,488 @@
+from collections import OrderedDict, defaultdict
+import importlib
+import inspect
+import json
+import json.decoder
+import six
+import warnings
+import sys
+
+import numpy as np
+import scipy.stats.distributions
+import sklearn.base
+import sklearn.model_selection
+# Necessary to have signature available in python 2.7
+from sklearn.utils.fixes import signature
+
+from .flow import OpenMLFlow
+
+
+if sys.version_info >= (3, 5):
+    from json.decoder import JSONDecodeError
+else:
+    JSONDecodeError = ValueError
+
+
+"""Convert scikit-learn estimators into an OpenMLFlows and vice versa."""
```

Comment:
If this is a module level docstring, I think it needs to be at the top of the file (pep8 will allow that)